packages/devtools_app/lib/src/profiler/cpu_profiler.dart

```diff
@@ -94,6 +94,9 @@ class _CpuProfilerState extends State<CpuProfiler>
     if (widget.tabs.length != oldWidget.tabs.length) {
       _initTabController();
     }
+    if (widget.data != oldWidget.data) {
+      setState(() {});
```

Comment:
OOC, what does calling `setState` with no data do? Trigger a refresh of the widget?

Reply:
setState will force the widget to be rebuilt. Calling an empty setState should be avoided though where possible, for this reason - it makes it hard to read and understand what state has changed and why we need to rebuild the widget. I violated best practices here, so I updated it to actually set a `data` var with the updated value we need to rebuild with.